src/backends/sqlite3/vector-into-type.cpp

```diff
@@ -132,8 +169,16 @@ void sqlite3_vector_into_type_backend::post_fetch(bool gotData, indicator * ind)
     int const endRow = static_cast<int>(statement_.dataCache_.size());
     for (int i = 0; i < endRow; ++i)
     {
-        sqlite3_column &col = statement_.dataCache_[i][position_-1];
-
+        std::shared_ptr<sqlite3_column> col1;
```

Comment:
Why do we use a shared pointer here? It doesn't seem to be shared with anybody...